### PR TITLE
Bring parity to the docker images on the install docs pages

### PR DESCRIPTION
**Pull request by davinci26**
Signed-off-by: Sotiris Nanopoulos <sonanopo@microsoft.com>
Commit Message:

As pointed out on the Windows sections we only linked to the dev images in `install.rst`. This should not be the case for official releases. We resolve it with using the `|envoy_windows_docker_image|` substitution block.

Risk Level: N/A docs only
Testing: N/A 
Docs Changes: Only docs
Release Notes: N/A 
Platform Specific Features: N/A

### CODE REVIEW

**Comment by davinci26** [comment]
cc @phlax and @envoyproxy/windows-dev

File: docs/root/start/install.rst [hunk 1]
```diff
@@ -135,10 +135,10 @@ Install Envoy on Windows
 
 You can run Envoy using the official Windows development Docker image.
```
Comment (pravb) [inline docs/root/start/install.rst:136]: Remove word "development"

**Review by pravb — DISMISSED** [review]

File: docs/root/start/install.rst (hunk 1 above)
Comment (davinci26) [reply docs/root/start/install.rst]: fixed in 8fc2acd074520e5aaf34b60591e715fc29db385e

**Review by phlax — APPROVED** [review]
lgtm, thanks @davinci26

**Review by asraa — COMMENTED** [review]
Do you want to add `envoy-windows` release image to the table below? https://github.com/envoyproxy/envoy/blob/8fc2acd074520e5aaf34b60591e715fc29db385e/docs/root/start/install.rst#pre-built-envoy-docker-images

**Comment by phlax** [comment]
+1 good spot - heres the (currently) rendered version https://storage.googleapis.com/envoy-pr/8fc2acd/docs/start/install.html#pre-built-envoy-docker-images

**Comment by davinci26** [comment]
good point, let me add that

**Comment by davinci26** [comment]
Just updated

**Review by asraa — APPROVED** [review]
Thanks!

**Pull request merged**